packages/roosterjs-editor-api/lib/utils/applyStyleToListItems.ts

```diff
@@ -0,0 +1,14 @@
+import { safeInstanceOf, setListItemStyle } from 'roosterjs-editor-dom';
+/**
+ * @internal
+ * Checks if the parent element is a List Item, if it is, apply the style elements to the list item
+ * @param parentNodes parentNodes to apply the style
+ * @param styles styles to apply to the List Item Element
+ */
+export function applyStyleToListItems(parentNodes: Node[], styles: string[]) {
```

Comment:
Then this file can be part of the wrapper, no need to have a separate file

Reply:
suggest move this function into applyListItemStyleWrap.ts and no need to export it.